### ISSUE TITLE
Incorrect list for "Humans on the ISS right now"

### ISSUE THREAD

**Issue by robbely** [mpurses/Sonder#20]
The list of "Humans on the ISS right now" includes the crew of the Shenzhou 12, which although currently in space, is not going to be docking with the ISS.

**Comment by mpurses** [comment]
Ya, I'm going to change it to Humans in space for the next version.

**Comment by robbely** [comment]
Yes that makes sense with more and more non-ISS space jaunts being planned.

I imagine there isn't any publicly accessible data to be able to add the Chinese space station to the map?
